### DISCUSSION TITLE
What to use for a clean Symfony dev docker configuration?

### DISCUSSION THREAD

**Discussion by jessybreyne** [php/frankenphp#93] [Q&A] [answered]
What is the difference betwen the `frankenphp` repository and the `dunglas/symfony-docker` repository?

I'm totally lost in the initialization of my docker for Symfony. Do you have projects example? 
I just need a mariadb service with my Symfony.

Does Symfony manage Docker stuff or do we have to do all the `docker compose`, ngnix or caddy, etc?

**Comment by dunglas** [comment, marked as answer]
Symfony Docker uses PHP FPM (for now). FrankenPHP is an alternative SAPI but is still experimental. `dunglas/frankenphp` contains this experimental SAPI/app server, `dunglas/frankenphp-demo` contains a FrankenPHP demo project using Symfony and `dunglas/symfony-docker` contains a Docker installer and runtime for Symfony that doesn't use FrankenPHP (at least for now).
